### ISSUE TITLE
Fix how responsibilities & duties apply to non-partner role-fillers

### ISSUE THREAD

**Issue by brianjrobertson** [holacracyone/Holacracy-Constitution#219]
Some responsibilities and duties were written assuming only Partners are Role-fillers, and either most should be broadened to apply to non-partner role-fillers as well, or non-Partners shouldn't be allowed as Role-Fillers (in which case another pathway is needed to have arms-length role-fillers who aren't invited to governance).

**Comment by karilen** [comment]
I thought a partner was anyone filling a role. I think you can just describe two kinds of partners, one subject to all the duties and one who is represented somehow else in the practice, like with outsourced or externals. The only thing needed is a brief description of different kinds of partners and a mechanism for making an external partner or whatever simply not invited by default, with the lead link repping the role, or some path for linking these arms length role fillers.

In some cases the contractors and arms length role fillers are invited to governance. Onboarding is needed there and there have been many success stories about this. I think clarity of how to interact if you're filling a role that is "more arms length" like the LL reps you, so maybe a new accountability on LL for representing any roles that are not de minimus but are not invited but external. Something like de minimus but with large focus and just not invited to gov, or subject to all the duties by default. I hesitate with non partner. And external doesn't make sense.

**Comment by ebabinet** [comment]
I definitely see a need to improve the constitution to allow for
"Non-Partner Role-fillers" or "Partners With Restricted Authority" (choose
your preferred point of view/implementation).

Need to clarify exactly what responsibilities/duties/authorities they have.
I think they should have the following:

   - they are not Role Representatives for their roles (at least by
   default, but Karilen's previous comment suggests that this is needed in
   some cases)
   - they cannot take Individual Action
   - they have all the duties of Transparency, Processing, and
   Prioritization
   - they have all the responsibilities of Role-Fillers

I don't know if it's the best way, but one relatively easy way to implement
this would be to define a new term (e.g. "External Partner" or "Limited
Partner") and to say that an External/Limited Partner is a Partner, except
where noted.

**Comment by karilen** [comment]
I wonder if this should include part time, as many orgs have part time employees, volunteers, or otherwise "internals" who may be in this category of not attending all the meetings and not being subject to all the same duties. Or maybe that is a different issue. Glad you mentioned it Eric Babinet to help me realize maybe that segment of people is not for this issue, not sure. Like your suggestions to get the ball rolling. I think that some of this non partner should not have all the duties E mentioned, but not sure what yet.

**Comment by ebabinet** [comment]
Yes, volunteers is definitely a use  which I would like this change to be
able to accommodate. Even for volunteers though, I think I would want all
the duties to apply, except perhaps "Requested Meetings Over Execution".
Karilen, I'd be curious to hear which duties you think are not needed.

**Comment by LouisChiquet** [comment]
So basically we're talking about [2.3](https://github.com/holacracyone/Holacracy-Constitution/blob/master/Holacracy-Constitution.md#23-duties-to-other-role-fillers)? Knowing before it wasn't “_As a Partner of the Organization_” but “_When filling a Role in a Circle_”, so just do that move back? And let all the newly amended part “_you have the following duties to others in your Organization, but only when they’re acting on behalf of one of their Roles and name that Role in any explicit request._”.
Would that solve it? Or one of the three might be too much if an external was in a role? 2.3.1 Duty of Transparency, 2.3.2 Duty of Processing, 2.3.3 Duty of Prioritization? We have a Circle in iGi where we have 3 externals in a Circle, they're just excluded from meetings and I haven't seen misprints.

For the individual actions, as it's precise “As a Partner of the Organization” in [2.6](https://github.com/holacracyone/Holacracy-Constitution/blob/master/Holacracy-Constitution.md#26-individual-action) it remains clear, so I don't believe there are many amendments needed on that part at least. Except for the 2.3, I don't see what doesn't work with that?

**Comment by ebabinet** [comment]
There would also need to be a change to the first sentence of 2.1
<https://github.com/holacracyone/Holacracy-Constitution/blob/master/Holacracy-Constitution.md#21-authority-of-role-fillers>
(ie. change it to "When filling a Role, you have the authority...") and to
the first sentence of 2.2
<https://github.com/holacracyone/Holacracy-Constitution/blob/master/Holacracy-Constitution.md#22-responsibilities-of-role-fillers>
(remove "As a Partner of the Organization", so the sentence starts "You
have the following responsibilities for each Role...")

**Comment by brianjrobertson** [comment]
I just went through this in-depth, along with another set of changes I'll submit soon (big update coming soon).  I did change the start of 2.1 per Eric's suggestion; but for 2.2 and 2.3, I left it specific only to partners.  The reason is that not all external contractors will necessarily have all of those responsibilities or duties, and it's useful to allow that; if the org wants the contractor to have them, they can simply add them to the contract, and done, they've got them.  But there are times where you don't want a contractor to, say, process accountabilities regularly but just execute certain projects instead.  H1 for example has an external bookkeeper contractor, and we couldn't assign the bookkeeper to a role right now if these duties applied, because not all of them are aligned with the contract or make sense; yet it's really useful to show the bookkeeper filling the role, because they are doing all of the accountabilities - just not necessarily with the same bar of role processing or duties to others as a partner would have.
